website_sale_product_brand/README.rst

```diff
@@ -33,21 +37,35 @@ Based on this configuration, you will see the menuitem shop by brand under shop
 It will show all the brands and once you select that brand it will show product's which
 is related to this brand.
 
+The blog here explains the HOWTO :
```

Comment:
Yes, please put here the content of the blog post. We want all the documentation auto-contained.